gamemodes/terrortown/entities/entities/ttt_radio.lua

```diff
@@ -286,7 +289,7 @@ local nextplay = 0
 ---
 -- @realm shared
 function ENT:Think()
-	if CurTime() <= nextplay or #self.SoundQueue <= 0 then return end
+	if CurTime() <= nextplay or (istable(self.SoundQueue) and #self.SoundQueue <= 0) then return end
```

Comment:
not sure if this change is needed. At some point I had an issue while hotreloading that was fixed with this change